lib/utils.js

```
@@ -0,0 +1,60 @@
+/**
+ * Lifted and tweaked from semantic-release because we follow how they bump their packages/dependencies.
+ * https://github.com/semantic-release/semantic-release/blob/master/lib/utils.js
+ */
+
+const { gt, lt, prerelease, rcompare } = require("semver");
+
+/**
+ * Get tag objects and convert them to a list of stringified versions.
+ * @param {array} tags Tags as object list.
+ * @returns {array} Tags as string list.
+ * @internal
+ */
+function tagsToVersions(tags) {
+	if (!tags) tags = [];
```

How about "fail fast"?
```js
if (!tags) {
  return []
}
```